app/views/mobile.scala

```diff
@@ -47,11 +47,9 @@ object mobile {
             ),
             div(cls := "right-side")(
               img(
-                cls := "nexus5-playing",
-                width := "268",
-                height := "513",
```

Comment:
`width` and `height` attributes should be specified whenever possible (and they are not the same as `width` and `height` in CSS, which are just styles). It helps the renderer to do layouting before the image is fully loaded.

Reply:
Sorry, by renderer, I mean the browser.

The `width` and `height` attribute are not inline styles. They specify the intrinsic size of the image, before styling. Suppose in CSS you specify only `width: 10em` (which really is a style), the corresponding height would be unknown until the image is downloaded and its intrinsic size is known. By specifying the intrinsic size, the browser can do layouting before the image is downloaded.

That's why it's best practice to always supply `width` and `height` attributes for `img` whenever they are known.

Reply:
Thank you for this explanation of what is supposed to be happening. Perhaps there is a bug in that case? This is what I am seeing in the browser in production:

<img width="1762" alt="Screenshot 2021-12-07 at 13 44 35" src="https://user-images.githubusercontent.com/11768967/145040443-1a6d8dff-c305-407e-94fb-48a5fa55a506.png">

Do you know the likely cause? If so, I will happily fix as part of this PR.